src/styles/components/_progress.scss

```diff
@@ -0,0 +1,47 @@
+$LINEAR_ANIMATION_TIME: 2.5s;
+
+@keyframes hold-visibility {
+  from { visibility: visible; }
+  to   { visibility: visible; }
+}
+
+@keyframes loading-progress-before {
+  from { transform: translate(-50%); }
+  to   { transform: translate(100vw) translate(-50%); }
+}
+
+.w-loading-progress {
+  background: #3fc4ff;
+  display: block !important;
+  height: 2px;
+  left: 0;
+  pointer-events: none;
+  position: fixed;
+  right: 0;
+  top: 0;
+  transform: scaleY(1);
+  transform-origin: 50% 0%;
+  transition: transform $TRANSITION_SPEED ease;
+  transition-delay: 1s;  // show loading hint only after waiting this long
+  z-index: 100000;
```

Comment:
Can you check the z-index for the side nav drawer? I was curious if this would put it above that.

Reply:
Yes. I've put it below. We should clean this up with Sass vars at some point.